### DISCUSSION TITLE
Kilosort-defined unit qualities

### DISCUSSION THREAD

**Discussion by twheatcroft07** [dandi/helpdesk#142] [Q&A]
I am interested in data from the Janelia brain-wide recording paper ([this](https://www.sciencedirect.com/science/article/pii/S0092867423014459)), for example, [ this file](https://neurosift.app/?p=/nwb&url=https://api.dandiarchive.org/api/assets/e1e626c9-f11e-4316-95b3-44c4f485d3ef/download/&dandisetId=000363&dandisetVersion=0.230822.0128). Unit quality, I believe, is based off of post-kilosort processing. I am interested in whether it is possible to read out the unit quality labels given from kilosort directly, before post-processing. Is this data available anywhere? Obviously I would prefer not to run kilosort again on each session.

Thanks!

**Comment by CodyCBakerPhD** [comment]
@alejoe91 Does SpikeInterface have the ability to recompute `good` vs. `mua` in the KiloSort/Phy sense without resorting or manually recurating the data?

**Reply by twheatcroft07** [reply]
Thankyou for investigating!

Since posting, I've come to the conclusion that 'unit_quality' might be the direct output of kilosort, as opposed to 'classification' which might be the output of post-kilosort processing. So I think I might have found the direct output of kilosort already.
